index.css

```
@@ -0,0 +1,107 @@
+* {
+    font-family: 'Courier New', Courier, monospace;
+    margin: 0px;
+}
+
+.header * {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+header {
+    display: inline;
+    width: 100vw;
+    font-size: 3.5rem;
+    align-items: center;
+    white-space: nowrap;
+}
+
+.div-header {
+    background-color: rgb(75, 167, 233);
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 5rem;            
+    padding: 0rem 3rem;
+}
+
+.info-perfil, p, b {
+    font-size: 1rem;
+    text-align: end;
+}
+
+.nav-header {
+    background-color: rgb(4, 63, 63);
+    height: 1.5rem;
+    font-size: 1.2rem;
+    display: flex;
+    align-items: center;            
+    padding: 0rem 3rem;
+    color: aliceblue;
+}
+
+.aside {
+    background-color: silver;
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    z-index: 2;
+    width: 16rem;
+    height: calc(100vh - 10rem);
+    margin-left: 3rem;
+    padding: .3rem 0rem;
+}
+
+input:focus {
+    outline: none;
+}
+
+.search-aside {
+    align-self: center;
+    width: 14.5rem;
+    margin-top: 1.5rem;
+    margin-bottom: .3rem;
+    font-size: 16px;
```

```suggestion
    font-size: 1rem;
```

es lo mismo (cuando nadie ha cambiado el tamanno por defecto es `16px`

---

jejeje esto no se como se me fue!!; si desde que vi tu clase no he hecho mas uso de px